### PR TITLE
Add Scale objects for Vegalite spec completion

### PR DESCRIPTION
**Pull request by aggFTW**
Added unit tests too

### CODE REVIEW

File: altair/api.py [hunk 1]
```diff
@@ -72,9 +72,31 @@ def to_dict(self):
...
+    # TODO: Supported types for type
+    # TODO: Supported types for reverse
+    # TODO: Supported types for zero
+    # TODO: Supported types for nice
+
+    type = T.Enum(['linear', 'log', 'pow', 'sqrt', 'quantile'], default_value='linear')
+    reverse = T.Bool(False)
+    zero = T.Bool(True)
+    nice = T.Enum(['second', 'minute', 'hour', 'day', 'week', 'month', 'year'], allow_none=True)
+    useRawDomain = T.Bool(default_value=None, allow_none=True)
```
Comment (ellisonbg) [inline altair/api.py]: One thing we will have to think about is if we want to convert the naming of things like `useRawDomain` to the more python style names of `use_raw_domain`. But lets leave it like this for now.

File: altair/api.py [hunk 2]
```diff
@@ -72,9 +72,31 @@ def to_dict(self):
...
 
-    pass
+
+class ScaleColor(Scale):
```
Comment (ellisonbg) [inline altair/api.py]: Do  they have this object in the spec? If not, I would `ColorScale` make more sense?

**Comment by ellisonbg** [comment]
Overall, this looks *great*. Just a few minor comments. Also looks like the tests were failing on python 2.7 and 3.4

File: altair/api.py (hunk 1 above)
Comment (aggFTW) [inline altair/api.py]: Let's keep that in mind.

File: altair/api.py (hunk 2 above)
Comment (aggFTW) [inline altair/api.py]: They don't. I'll change it to ColorScale.

**Comment by aggFTW** [comment]
The tests were failing? The Travis build shows them as passing. Am I missing something?

**Comment by ellisonbg** [comment]
Hmm, I did restart them - sometimes Travis needs to be kicked...

On Wed, Oct 14, 2015 at 9:53 PM, Alejandro Guerrero Gonzalez <
notifications@github.com> wrote:

> The tests were failing? The Travis build shows them as passing. Am I
> missing something?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ellisonbg/altair/pull/69#issuecomment-148254318>.
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by aggFTW** [comment]
Oh :) I didn't know that haha
Well, they are passing now. Let me know if there's any more changes we need to make before this PR can be merged.

**Comment by ellisonbg** [comment]
Looks great, merging!

**Pull request merged**